sql/sql_acl.cc

```diff
@@ -3609,7 +3609,7 @@ bool check_some_access(THD *thd, privilege_t want_access, TABLE_LIST *table)
     1	Access denied.  In this case an error is sent to the client
 */
 
-bool check_global_access(THD *thd, privilege_t want_access, bool no_errors)
+bool check_global_access(THD *thd, const privilege_t want_access, bool no_errors)
```

Comment:
I don't get it. in the previous commit all prototypes of `check_global_access` don't have `const`, where is this one coming from?